ccdproc/ccddata.py

```diff
@@ -189,14 +189,28 @@ def uncertainty(self, value):
         else:
             self._uncertainty = value
 
-    def to_hdu(self):
+    def to_hdu(self, hdu_mask='MASK', hdu_uncertainty='UNCERT',
+               hdu_flags=None):
         """Creates an HDUList object from a CCDData object.
 
+           Parameters
+           ----------
+           hdu_mask, hdu_uncertainty, hdu_flags : str or None, optional
+              If it is a string append this attribute to the HDUList as
+              `~astropy.io.fits.ImageHDU` with the string as extension name.
+              Flags are not supported at this time. If ``None`` this attribute
+              is not appended.
+              Default is ``'MASK'`` for mask, ``'UNCERT'`` for uncertainty and
+              ``None`` for flags.
+
            Raises
            -------
            ValueError
```

Comment:
Actually this could be removed as it isn't relevant here, but the ValueErrors rased for bad mask or uncertainty frames should be mentioned here